tox.ini

```diff
@@ -31,6 +31,20 @@ basepython = python3.4
 deps = flake8>=2.4,<2.5
 commands = flake8 glitter
 
+[testenv:docs]
+skip_install = true
+deps =
+    -rdocs/requirements.txt
+    restructuredtext_lint>=0.14,<0.15
+    doc8>=0.7.0,<0.8.0
+commands =
+    rst-lint README.rst
+    doc8 docs/source
+    sphinx-build -W -a -b html docs/source docs/build
+
 [flake8]
 max-line-length = 100
```

Comment:
Would prefer to not touch code with this branch, you'll be bringing this in with add-tags anyway.